meshtastic/node_runtime/seturl/coordinator.py

```
@@ -0,0 +1,130 @@
+from meshtastic.node_runtime.seturl.rollback import _SetUrlRollbackEngine
+
+if TYPE_CHECKING:
+    from meshtastic.node import Node
+
+logger = logging.getLogger(__name__)
+
+
+class _SetUrlTransactionCoordinator:
+    """Coordinates setURL transaction planning, execution, rollback, and cache policy."""
+
+    def __init__(self, node: "Node", *, parsed_input: _SetUrlParsedInput) -> None:
```

Ruff (0.15.7)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)